lib/cocoapods/generator/xcconfig/aggregate_xcconfig.rb

```diff
@@ -94,15 +95,11 @@ def generate
         #
         def settings_to_import_pod_targets
           if target.requires_frameworks?
-            framework_header_search_paths = pod_targets.select(&:should_build?).map do |target|
-              if target.scoped?
-                "$PODS_FRAMEWORK_BUILD_PATH/#{target.product_name}/Headers"
-              else
-                "$CONFIGURATION_BUILD_DIR/#{target.product_name}/Headers"
-              end
+            build_pod_targets = pod_targets.select(&:should_build?)
+            framework_header_search_paths = build_pod_targets.map do |target|
+              "#{target.configuration_build_dir}/#{target.product_name}/Headers"
```

Comment:
Why? It's just used at this one place. I can't recognize any advantage of having a quick accessor for it.